.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: ci
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
Actually can we also add `8.x` here ? That's the minimal version we support.

Reply:
Yep sure I'll add it. But just FYI, React Native is in the process of dropping 8.x (the version is also no longer officially supported, maintenance ended at the end of 2019). For example one commit that got pushed to `master` today: https://github.com/facebook/react-native/commit/b53aeab10cecfa9b08879b846f8e33d41b28c898
But I'm OK adding it here for now, and we can consider removing it once we move up to the latest version of RN.

Reply:
https://nodejs.org/en/about/releases/